internal/hack/hack.go

```diff
@@ -0,0 +1,7 @@
+package hack
```

Comment:
Suggest to move this to pkg/cast package.

Reply:
done